app/celery/scheduled_tasks.py

```diff
@@ -173,8 +175,8 @@ def check_job_status():
 @notify_celery.task(name='replay-created-notifications')
 @statsd(namespace="tasks")
 def replay_created_notifications():
-    # if the notification has not be send after 4 hours + 15 minutes, then try to resend.
-    resend_created_notifications_older_than = (60 * 60 * 4) + (60 * 15)
+    # if the notification has not be send after 1 hour, then try to resend.
```

Comment:
We are OK to resend emails and SMS after an hour then I take it?

Reply:
Yes - the task to send email/SMS can be called twice since it checks the notifications status. You shouldn't get the email/SMS sent twice.